apps/accounts/models.py

```
@@ -1,23 +1,33 @@
+from django.contrib.auth.models import PermissionsMixin
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
 from .managers import CustomUserManager
 
 
-class User(AbstractUser):
+class User(AbstractBaseUser, PermissionsMixin):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    username = None
+    display_name = models.CharField(_("display name"), max_length=255, null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)